internal/provider/resource_gitlab_topic.go

```diff
@@ -32,21 +33,48 @@ var _ = registerResource("gitlab_topic", func() *schema.Resource {
 
 		Schema: map[string]*schema.Schema{
 			"name": {
-				Description: "The topic's name",
+				Description: "The topic's name.",
 				Type:        schema.TypeString,
 				Required:    true,
 			},
 			"soft_destroy": {
-				Description: "Empty the topics fields instead of deleting it",
+				Description: "Empty the topics fields instead of deleting it.",
 				Type:        schema.TypeBool,
 				Optional:    true,
 				Deprecated:  "GitLab 14.9 introduced the proper deletion of topics. This field is no longer needed.",
 			},
 			"description": {
-				Description: "A text describing the topic",
+				Description: "A text describing the topic.",
 				Type:        schema.TypeString,
 				Optional:    true,
 			},
+			"avatar": {
+				Description: "A local path to the avatar image to upload. **Note**: not available for imported resources",
```

Comment:
```suggestion
				Description: "A local path to the avatar image to upload. **Note**: not available for imported resources.",
```
I see you're adding periods to others, but don't have one at the end here 😉

Reply:
thanks!